packages/router/.gitignore

```diff
@@ -61,4 +61,7 @@ typings/
 .next
 
 # WebStorm
-.idea
\ No newline at end of file
+.idea
```

Comment:
I think the `.gitignore` in sub package should be removed. it won't work, right ?

Reply:
@kunyan It'd work. It will simply override the root `.gitignore`.

Reply:
I'll remove it from the sub package

Reply:
fixed